transport/src/test/java/io/zeebe/transport/impl/AtomixTransportTest.java

```diff
@@ -283,16 +289,17 @@ public void shouldRetryAndSucceedAfterResponseIsValid() throws InterruptedExcept
     // given
     final var retryLatch = new CountDownLatch(2);
     serverTransport.subscribe(0, new DirectlyResponder(bytes -> retryLatch.countDown())).join();
+
     final var responseValidation = new AtomicBoolean(false);
     final var requestFuture =
         clientTransport.sendRequestWithRetry(
             nodeAddressSupplier,
             (responseToValidate) -> responseValidation.get(),
             new Request("messageABC"),
-            Duration.ofSeconds(3));
```

Comment:
I would like to keep the request timeouts higher

Reply:
You would like to wait for more than 10 seconds? What do you suggest? :wink:

Reply:
My comment was about higher request timeout then latch timeout